src/pages/ViewTeamGrades/ViewTeamGrades.module.scss

```
@@ -84,40 +84,43 @@
 }
 
 /* Colors used for coloring score cells within the heatgrid for the grades view */
+/* These classes are returned as plain strings by getColorClass() and used globally
+   in FeedbackTable.tsx and ReviewTableRow.tsx, so they must not be locally scoped. */
+:global {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)